util/src/main/java/io/zeebe/util/sched/ActorThread.java

```diff
@@ -32,13 +32,16 @@
 import org.agrona.UnsafeAccess;
 import org.agrona.concurrent.BackoffIdleStrategy;
 import org.agrona.concurrent.ManyToManyConcurrentArrayQueue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.slf4j.MDC;
 import sun.misc.Unsafe;
 
 @SuppressWarnings("restriction")
 public class ActorThread extends Thread implements Consumer<Runnable> {
   static final Unsafe UNSAFE = UnsafeAccess.UNSAFE;
   private static final long STATE_OFFSET;
+  private static final Logger LOG = LoggerFactory.getLogger(ActorThread.class);
```

Comment:
Shouldn't we use the existing `ActorLogger`?
https://github.com/zeebe-io/zeebe/blob/develop/util/src/main/java/io/zeebe/util/Loggers.java#L16

Reply:
Yes; why I initially went with a per-class logger is because I think in the long run it makes sense for us to drop these broad loggers. If you want to get fine-grained trace logs, or debug, etc., if you want control over what you're seeing, having one logger used for a complete unit is counter productive as you may get too much logs. At the same time, having one logger per class is cheap, and you can still group things by packages...

However for this PR we probably should remain consistent, and moving away from module-level loggers or some such should be something we do in a separate PR.

Reply:
I agree. Let's discuss/announce it in the team meeting first :)